jira/client.py

```diff
@@ -3984,7 +4043,18 @@ def permissionschemes(self):
         return data["permissionSchemes"]
 
     @lru_cache(maxsize=None)
-    def issuesecurityschemes(self):
+    def issue_type_schemes(self) -> List[IssueTypeScheme]:
+        """get all issue type schemes defined"""
+
+        url = self._get_url("issuetypescheme")
+
+        r = self._session.get(url)
+        data: Dict[str, Any] = json_loads(r)
+
+        return data["schemes"]
+
+    @lru_cache(maxsize=None)
+    def issue_securityschemes(self):
```

Comment:
```suggestion
    def issuesecurityschemes(self):
```

is the original name (unfortunately)

Reply:
whoops, good catch